dev-manuals/backend/Libraries/AllService_libraries.md

```diff
@@ -0,0 +1,52 @@
+|                 | Gits Common Test Library             | 0.1.0     | Contains common utilities and functionality for testing GITS projects.                           |
+|                 | JUnit                                | 4.13.1    | Testing framework for unit tests.                                                                |
+|                 | Mockito                              | 3.+       | Framework for creating mock objects in unit tests.                                               |
+|                 | Hamcrest                             | 2.+       | Framework for writing matchers in test assertions.                                               |
+|                 | Testcontainers PostgreSQL            | 1.18.3    | Docker-based PostgreSQL container for integration testing.                                       |
+|                 | Testcontainers Junit Jupiter         | 1.18.3    | Integration between Testcontainers and Junit Jupiter for testing.                                |
+
+<div style="text-align: right;">
+
+<small>**All Services** include skill level, quiz service, content, and course service.</small>
+</div>
+
+| Service                | Library                              | Version   | Purpose                                                              |
+|:-----------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Media Service**      | Minio SDK                            | 8.5.2     | Provides a client for interacting with the Minio object storage server. |
+
+| Service                | Library                              | Version   | Purpose                                                              |
+|:-----------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Content Service**    | H2 Database                          |           | In-memory database for local development and testing.                |
+
+| Service                | Library                              | Version   | Purpose                                                              |
+|:-----------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Course Service**     | JSR 305                              | 3.0.2     | Removes a Gradle warning about an unknown annotation.                |
+
+| Service                | Library                              | Version   | Purpose                                                              |
+|:-----------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Reward Service**     | Content Service                      | 0.1.2     | Includes content service for DTOs.                                   |
```

Comment:
Doesn't the skill level service also contain the content service as dependeny?

Reply:
yes, I forgot sorry, I added